python/3.9-canine-util/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9
+LABEL maintainer="Daniel Miller (millerd15@chop.edu)"
+LABEL description="Python3 base with all necessary packages installed. Includes bedtools for pybedtools integration."
+
+RUN python -m pip install --upgrade pip && python -m pip install pandas pysam pybedtools pyyaml
```

Comment:
Why not env first, then RUN and && everything else?

Reply:
So the way docker works: if you set ENV before the pip install, if you were to update the env variable it would rerun the pip install since that's how layering works. In this example if you were to update the env variable it would use the existing layer as is and build on top of it. It's not hugely relevant to what we do since we tend to not really update existing dockers but I think its better practice.